ChakraCore Samples/JSRT Hosting Samples/C#/ChakraCoreHost/js/SetTimeoutTest1.js

```diff
@@ -0,0 +1,6 @@
+﻿host.echo("Set Timeout Test 1");
+function onTimeout() {
+    host.echo("onTimeout called");
+}
+
+setTimeout(onTimeout, 3000);
```

Comment:
Can you add another test of calling setTimeout within a setTimeout? I think it works but good to verify